articles/custom_storage_location.md

```diff
@@ -118,6 +123,62 @@ In **Permissions**, click **CORS configuration**. In the CORS configuration edit
 <br/>
 For more information, please read: [How Do I Configure CORS on My Bucket?](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors){:target="_blank"}
 
+
+### Setup with AWS Cloudformation
+
+For convienance [AWS Cloudformation] can be used to provision a custom AWS S3 bucket for use with Synapse.
+Using this approach will result in the exact same bucket as described in [Setup with AWS Console](#setup-with-aws-console).
+
+Instructions:
+1. Download the [CF template](https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/SynapseExternalBucket-v2.yaml).
```

Comment:
We should make a copy of this script and add it to the docs instead providing  a reference to scicomp because the one in scicomp may disappear one day.  it's actually very, very likely to disappear.

Reply:
Looks like it already disappeared. Do you still have a copy of the script?

Reply:
Or we could document the process of how someone finds the right version of this file/script?